.github/workflows/tests.yml

```diff
@@ -82,7 +82,7 @@ jobs:
           name: bazel-testlogs
           path: bazel-testlogs
 
-  lts_ios_integration_tests:
+  bazel_lts_ios_integration_tests:
```

Comment:
IMO the name here is confusing, if it's going to change in this PR could we add bazel5 to the name instead of LTS. 

LTS applies to multiple bazel versions including 6

Reply:
i think adding `bazel5` like i said would it make clear this specifically is a bazel5 job.

Reply:
+1 on being more explicit on the naming here

Reply:
Ok sounds good to me - this is a good improvement in the github UI